app/assets/javascripts/users.js

```diff
@@ -2,7 +2,14 @@
   "use strict";
   App.Users = {
     initialize: function() {
+      var observer;
       $(".initialjs-avatar").initial();
+      observer = new MutationObserver(function(mutations) {
+        $.each(mutations, function(index, mutation) {
+          $(".initialjs-avatar", $(mutation.addedNodes)).initial();
```

Comment:
This is the style I use instinctively. However, I noticed in some places we use a different style which might be more readable :thinking::

```javascript
$(mutation.addedNodes).find(".initialjs-avatar").initial();
```

Whichever style you prefer is fine with me :smile:.